### PR TITLE
Add content about Radius versioning in the scope of public release

### PR DESCRIPTION
**Pull request by Reshrahim**
Thank you for helping make the Radius documentation better!

**Please follow this checklist before submitting:**

- [ ] [Read the contribution guide](https://docs.radapp.dev/contributing/docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description
Adding content about Radius versioning in the scope of public release that we will be introducing breaking changes.

### Summary
📝🚧🔢

Add a new document `versioning.md` to explain the current beta status and lack of backward compatibility of Radius. The document is part of a larger effort to improve the documentation quality and user experience.

> _`Radius` in beta_
> _No backward compatibility_
> _Fresh install each spring_

### Walkthrough
*  Add a new document about the versioning policy of Radius ([link](https://github.com/project-radius/docs/pull/633/files?diff=unified&w=0#diff-f754263958cf60864d2a90a205bf1d34b1b975d2b0431dd20ed8ab831e1a5f2bR1-R13))
*  Explain that Radius is in beta and not backward compatible, and advise users to do a fresh installation of the latest version ([link](https://github.com/project-radius/docs/pull/633/files?diff=unified&w=0#diff-f754263958cf60864d2a90a205bf1d34b1b975d2b0431dd20ed8ab831e1a5f2bR1-R13))
*  State that a versioning strategy is being worked on and will be announced in the future ([link](https://github.com/project-radius/docs/pull/633/files?diff=unified&w=0#diff-f754263958cf60864d2a90a205bf1d34b1b975d2b0431dd20ed8ab831e1a5f2bR1-R13))
*  Add the necessary metadata for the document, such as the type, title, linkTitle, description, and weight ([link](https://github.com/project-radius/docs/pull/633/files?diff=unified&w=0#diff-f754263958cf60864d2a90a205bf1d34b1b975d2b0431dd20ed8ab831e1a5f2bR1-R13))

### CODE REVIEW

File: docs/content/operations/versioning.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+type: docs
+title: "Versioning"
+linkTitle: "Versioning"
+description: "Radius versioning policy"
+weight: 600
+---
+
+## Overview
+
+Radius is currently in beta and is not backward compatible with previous releases. Radius introduces breaking changes between releases and we recommend doing a fresh installation of the latest version of Radius everytime after a release.
```
Comment (AaronCrawfis) [inline docs/content/operations/versioning.md]:
```suggestion
Radius is currently in an early alpha preview and does not offer backward compatibility with previous releases. Breaking changes may happen between releases and we recommend doing a fresh installation of the latest version of Radius after every release.
```

File: docs/content/operations/versioning.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+We are working on the versioning strategy and will be introducing a versioning policy in the future.
```
Comment (AaronCrawfis) [inline docs/content/operations/versioning.md:14]:
```suggestion
A versioning strategy will be introduced prior to the launch and announcement of Radius. Stay tuned for updates.

```

File: docs/content/operations/versioning.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+type: docs
+title: "Versioning"
```
Comment (AaronCrawfis) [inline docs/content/operations/versioning.md]:
```suggestion
title: "Overview: Radius versioning policy"
```

File: docs/content/operations/versioning.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+linkTitle: "Versioning"
+description: "Radius versioning policy"
+weight: 600
+---
```
Comment (AaronCrawfis) [inline docs/content/operations/versioning.md:8]:
```suggestion
category: "Overview"
---
```

File: docs/content/operations/versioning.md (hunk 2 above)
Comment (Reshrahim) [reply docs/content/operations/versioning.md:14]: we do not have this scoped for public release. should we refrain from mentioning about launch/announcement?

Comment (AaronCrawfis) [reply docs/content/operations/versioning.md:14]: Good catch. Let's keep it as is

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**